lens/pom.xml

```diff
@@ -226,4 +226,15 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>vendor-repo</id>
+      <repositories>
+        <repository>
+          <id>cloudera</id>
+          <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
```

Comment:
Same question again - does R interpreter requires modification of `lets` module?

It will take a lot of time to review, as now we need somebody who maintain lens part to check it out.